### ISSUE TITLE
Error using hydra-core 1.0

### ISSUE THREAD

**Issue by yhangchen** [denisyarats/proto#2]
By installing hydra by
`pip install hydra-core --upgrade
`The following error arises:
```
Key 'name' not in 'HydraConf'
        full_key: hydra.name
        reference_type=Optional[HydraConf]
        object_type=HydraConf
```

**Comment by yhangchen** [comment]
sorry I found that it should be under env proto

**Comment by MisakaGeek** [comment]
> sorry I found that it should be under env proto

I encountered the same problem, could you elaborate on how to solve it?
Thanks!

**Comment by famishedrover** [comment]
Did you find a solution to this?

**Comment by haoruili699** [comment]
You may fix it by running: pip install hydra-core==0.11.3. This works for me.
